extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -31,146 +32,183 @@ if (context.RunTest) {
-				// script generation might take too long and the 'success' status does not mean that script is created.
-				await assertScriptGenerationResult(status);
-			}
-			finally {
-				await utils.deleteDB(sourceDB, ownerUri);
-				await utils.deleteDB(targetDB, ownerUri);
-			}
-		});
+			assert(schemaCompareService, 'Schema Compare Service Provider is not available');
 
-		test('Schema compare dacpac to database comparison and script generation', async function () {
-			let server = await getStandaloneServer();
-			await utils.connectToServer(server, SERVER_CONNECTION_TIMEOUT);
-
-			let nodes = <azdata.objectexplorer.ObjectExplorerNode[]>await azdata.objectexplorer.getActiveConnectionNodes();
-			assert(nodes.length > 0, `Expecting at least one active connection, actual: ${nodes.length}`);
-
-			let index = nodes.findIndex(node => node.nodePath.includes(server.serverName));
-			assert(index !== -1, `Failed to find server: "${server.serverName}" in OE tree`);
-
-			let ownerUri = await azdata.connection.getUriForConnection(nodes[index].connectionId);
-			let now = new Date();
-			let targetDB: string = 'ads_schemaCompare_targetDB_' + now.getTime().toString();
-
-			try {
-				let dacfxService = await azdata.dataprotocol.getProvider<azdata.DacFxServicesProvider>('MSSQL', azdata.DataProviderType.DacFxServicesProvider);
-				assert(dacfxService, 'DacFx Service Provider is not available');
-				let result = await dacfxService.deployDacpac(path.join(__dirname, 'testData/Database2.dacpac'), targetDB, true, ownerUri, azdata.TaskExecutionMode.execute);
-
-				assert(result.success === true, 'Deploy database 2 (target) should succeed');
-
-				let source: azdata.SchemaCompareEndpointInfo = {
-					endpointType: azdata.SchemaCompareEndpointType.Dacpac,
-					packageFilePath: dacpac1,
-					serverName: '',
-					databaseName: '',
-					ownerUri: ownerUri,
-				};
-				let target: azdata.SchemaCompareEndpointInfo = {
-					endpointType: azdata.SchemaCompareEndpointType.Database,
-					packageFilePath: '',
-					serverName: server.serverName,
-					databaseName: targetDB,
-					ownerUri: ownerUri,
-				};
-
-				assert(schemaCompareService, 'Schema Compare Service Provider is not available');
-				let schemaCompareResult = await schemaCompareService.schemaCompare(source, target, azdata.TaskExecutionMode.execute, null);
-				assertSchemaCompareResult(schemaCompareResult);
-
-				let status = await schemaCompareService.schemaCompareGenerateScript(schemaCompareResult.operationId, server.serverName, dummyDBName, azdata.TaskExecutionMode.script);
-				await assertScriptGenerationResult(status);
-			}
-			finally {
-				await utils.deleteDB(targetDB, ownerUri);
+			// start schema compare and cancel
+			const compareTask = schemaCompareService.schemaCompare('testOperationId', source, target, azdata.TaskExecutionMode.execute, null);
+			const canceResult = await schemaCompareService.schemaCompareCancel('testOperationId');
```

Comment:
nit typo - canceResult is missing an L

Reply:
ouch sorry.

Reply:
suggestion: use vscode extensions like: Code Spell Checker or Spell Right.